docs/cockroachdb.md

```diff
@@ -0,0 +1,300 @@
+---
+id: cockroachdb
+title: OpenEBS for CockroachDB
+sidebar_label: CockroachDB
+---
+------
+
+<img src="/docs/assets/o-cockroachdb.png" alt="OpenEBS and CockroachDB" style="width:400px;">
+
+
+## Introduction
+
+CockroachDB is a cloud-native SQL database for building global, scalable cloud services that survive disasters. It is a distributed SQL database built on a transactional and strongly-consistent key-value store. It scales horizontally; survives disk, machine, rack, and even datacenter failures with minimal latency disruption and no manual intervention; supports strongly-consistent ACID transactions; and provides a familiar SQL API for structuring, manipulating, and querying data.
+
+This guide explains the basic installation for CockroachDB operator on OpenEBS Local PV devices.
+
+
+<br>
+
+
+## Deployment model 
+
+<br>
+
+<img src="/docs/assets/svg/Local-PV-Distributed-device-cockroachdb.svg" alt="OpenEBS and CockroachDB" style="width:100%;">
+
+<br>
+
+The Local PV volume will be provisioned on a node where CockroachDB pods are getting scheduled and uses one of the matching unclaimed block device for each of them, which will then use the entire block device for storing data. No other application can use this device. If users have limited blockdevices attached to some nodes, they can use `nodeSelector` in the application YAML to provision applications on particular nodes where the available block device is present. The recommended configuration is to have at least three nodes and one unclaimed external disk to be attached per node. 
+
+
+## Configuration workflow 
+
+1. [Install OpenEBS](/docs/next/cockroachdb.html#instal-openebs)
```

Comment:
Hyperlink is still not correct.. Need to update the following as the correct link. 
/docs/next/cockroachdb.html#install-openebs

Reply:
thank you @ranjithwingrider for finding it out.